### PR TITLE
Add ui for toggle all

### PR DESCRIPTION
**Pull request by hyperwidget**
This PR does two main things:

* Includes linkedin in the `toggle all` functionality (it was previously erroneously not there)
* Adds a ui treatment for users to toggle all apps, and includes a tip for the quick keys combo

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,23 +28,27 @@
...
-                    Go
-                </button>
+                    <select name="additionalOptions" id="additional-options">
+                        <option value="feedback">Provide Feedback</option>
+                        <option value="additionalResource">Additional Resource</option>
+                        <option value="tweetUs">Tweet us!</option>
+                    </select>
+                    <button id="actionButton">
+                        Go
+                    </button>
```
Comment (hyperwidget) [inline index.html:45]: weird autoformats, but ok

File: src/contentScripts/myscript.js [hunk 2]
```diff
@@ -98,5 +96,7 @@ export const messageListener = chrome.runtime.onMessage.addListener(function(
     case request.toggleMeetup !== undefined:
       meetupUpdater(request.toggleMeetup)
       break
+    case request.toggleAll !== undefined:
+      break
```
Comment (hyperwidget) [inline src/contentScripts/myscript.js:100]: will remove

File: src/ui/index.js [hunk 3]
```diff
@@ -9,14 +9,16 @@ document.addEventListener('DOMContentLoaded', function() {
   var $toggleLever = $('#toggle-lever')
   var $toggleGithub = $('#toggle-github')
   var $toggleMeetup = $('#toggle-meetup')
+  var $toggleAll = $('#toggle-all')
 
-  const LINKEDIN = 'linkedin'
+  const LINKEDIN = 'linkedIn'
```
Comment (hyperwidget) [inline src/ui/index.js:14]: This was why linkedin toggle wasn't flipping when we toggled all

File: src/ui/index.js [hunk 4]
```diff
@@ -61,12 +64,27 @@ document.addEventListener('DOMContentLoaded', function() {
   })
 
   $toggleMeetup.off().on('change', function(e) {
-    const set = $toggleGreenhouse.is(':checked')
+    const set = $toggleMeetup.is(':checked')
```
Comment (hyperwidget) [inline src/ui/index.js:67]: lol, whoops

File: src/ui/index.js [hunk 5]
```diff
@@ -61,12 +64,27 @@ document.addEventListener('DOMContentLoaded', function() {
...
+    const set = $toggleAll.is(':checked')
+
+    chrome.storage.sync.set({ [ALL]: set })
+    $toggleLinkedIn.prop('checked', set).trigger('change')
+    $toggleMeetup.prop('checked', set).trigger('change')
+    $toggleAngelList.prop('checked', set).trigger('change')
+    $toggleLever.prop('checked', set).trigger('change')
+    $toggleGreenhouse.prop('checked', set).trigger('change')
+    $toggleGithub.prop('checked', set).trigger('change')
+    $toggleReplit.prop('checked', set).trigger('change')
```
Comment (hyperwidget) [inline src/ui/index.js:83]: There is likely a more streamlined/efficient way to do this, but I'm aight with this for now

**Review by bhavyata9 — APPROVED** [review]
LGTM!

**Pull request merged**